src/actions/ws_message.rs

```diff
@@ -0,0 +1,109 @@
+use std::collections::HashMap;
+
+use yaml_rust::Yaml;
+use colored::*;
+use serde_json;
+use time;
+
+use ws::{connect, CloseCode};
+
+use interpolator;
+use config;
+
+use actions::{Runnable, Report};
+
+#[derive(Clone)]
+pub struct WSMessage {
+  name: String,
+  url: String,
+  time: f64,
+  data: Option<String>,
+  pub with_item: Option<Yaml>,
+  pub assign: Option<String>,
+}
+
+impl WSMessage {
+  pub fn is_that_you(item: &Yaml) -> bool{
+    item["websocket_message"].as_hash().is_some()
```

Comment:
What about rename `websocket_message` to `websocket` and `data` to `message`? It feels simpler api.

Reply:
Good point, done.